gsa/src/web/pages/reports/details/emptyresultsreport.js

```diff
@@ -43,7 +43,7 @@ const FilterString = styled.span`
 `;
 
 const EmptyResultsReport = ({
-  all,
+  all, // 'all' is not used right now because it refers to the no. of rows in the results table. Should be re-implemented in future versions.
```

Comment:
I think since we do not use "all" anymore we should remove it completely. We don't know when we are going to re-implement it and since the actual implementation will probably be in the resultstab the prop we give to the emptyresultsreport might not even be called "all" then. If we don't use it at the moment it should not be in here.